.github/workflows/php_tests.yml

```diff
@@ -53,21 +52,34 @@ jobs:
         wp-version: ${{ fromJSON(needs.find_latest_versions.outputs.wp-versions) }}
         wc-version: ${{ fromJSON(needs.find_latest_versions.outputs.wc-versions) }}
     steps:
-      - name: Checkout latest WCS
+      - name: Checkout WCS&T
         uses: actions/checkout@v2
         with:
           submodules: recursive
-      - name: Switch to PHP 8.1
-        run: |
-          sudo update-alternatives --set php /usr/bin/php8.1
-          sudo update-alternatives --set phar /usr/bin/phar8.1
-          sudo update-alternatives --set phar.phar /usr/bin/phar.phar8.1
-      - name: Install composer
+      - name: Checkout WooCommerce
+        run: git clone --depth=1 --branch=${{ matrix.wc-version }} https://github.com/woocommerce/woocommerce.git /tmp/woocommerce
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+      - name: Setup PNPM
+        uses: pnpm/action-setup@c3b53f6a16e57305370b4ae5a540c2077a1d50dd
+        with:
+          version: '^7.22.0'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'v16'
+      - name: Build WooCommerce essentials for running tests
         run: |
-          ./bin/composer-installer.sh
-          sudo mv composer.phar /usr/local/bin/composer
+          cd /tmp/woocommerce/plugins/woocommerce
           composer install
-      - name: Setup WP and WC
-        run: tests/bin/install-wc-tests.sh wordpress_test root root 127.0.0.1 ${{ matrix.wp-version }} ${{ matrix.wc-version }}
+          pnpm run build:feature-config
+      - name: Setup WordPress
+        run: bash /tmp/woocommerce/plugins/woocommerce/tests/bin/install.sh wordpress_test root root 127.0.0.1 ${{ matrix.wp-version }}
+      - name: Use older PHPUnit for WC < 7.4.0
+        if: contains(fromJSON('["7.3.0", "7.2.3"]'), matrix.wc-version)
+        run: composer require -W phpunit/phpunit:^7
```

Comment:
If we use WC < 7.4.0 and PHPUnit 8, we run into errors about static fields not being declared.